Sources/Utils/Rng/MobileCoinRng.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+// swiftlint:disable unavailable_function
+public class MobileCoinRng {
```

Comment:
what about making this implement the Swift standard protocol `RandomNumberGenerator` and renaming the nextUInt64() to next(), the return types are the same.

Reply:
Sounds good - will do.

Reply:
code updated.